sa/database.go

```diff
@@ -11,30 +11,50 @@ import (
 	"net/url"
 	"strings"
 
-	// Provide access to the MySQL driver
-	_ "github.com/letsencrypt/boulder/Godeps/_workspace/src/github.com/go-sql-driver/mysql"
+	mysql "github.com/letsencrypt/boulder/Godeps/_workspace/src/github.com/go-sql-driver/mysql"
```

Comment:
nit: naming this import is unnecessary since the package is already named `mysql`